sources/au/victoria.json

```diff
@@ -1,7 +1,7 @@
 {
     "type": "http",
-    "data": "http://s3.amazonaws.com/data.openaddresses.io/cache/au-victoria.zip",
-    "website": "http://services.land.vic.gov.au/landchannel/content/vicmapdata?productID=1",
+    "data": "https://www.alantgeo.com.au/share/au-victoria.zip",
```

Comment:
Thanks for pointing that out, I forgot that file was still needed for this PR!

I'm re-uploading it now, will post back when it's done.

It's the same as downloading via the official website (except the official website requires you to go through a shopping cart process and the download URL isn't static).